src/config.js

```diff
@@ -1,6 +1,7 @@
-const { REQUEST_THROTTLE_MS, REQUEST_TIMEOUT_MS } = process.env
+const { DIFFUSION_REQUEST_THROTTLE_MS, REQUEST_THROTTLE_MS, REQUEST_TIMEOUT_MS } = process.env
 
 export default {
+  DIFFUSION_REQUEST_THROTTLE_MS: Number(DIFFUSION_REQUEST_THROTTLE_MS) || 600000,
   REQUEST_THROTTLE_MS: Number(REQUEST_THROTTLE_MS) || 5000,
```

Comment:
The default of 5 seconds is pretty generous, maaaybe we should increase it?

Aside- we haven't ever set this in production so we're still on that relatively fast refresh. Might we want to adjust that a bit (maybe it'll help with the memory woes)?

Reply:
I set the default for diffusion to be 10 minutes.